src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -127,6 +127,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@adityapatwardhan, your last commit had 39 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Remove-Item UnAuthorized Access.Access-denied test for removing a folder
```powershell
Cannot validate argument on parameter 'Path'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\FileSystem.Tests.ps1: line 1410
1410:         $acl = Get-Acl $protectedPath

```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable gets all compatible modules when SkipEditionCheck: $false, using root module: $true, using absolute path: $false, CompatiblePSEditions: $nu
```powershell
Expected 0, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 493
493:                 $modules.Count | Should -Be 0
```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable gets all compatible modules when SkipEditionCheck: $false, using root module: $true, using absolute path: $false, CompatiblePSEditions: Des
```powershell
Expected 0, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 493
493:                 $modules.Count | Should -Be 0
```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable gets all compatible modules when SkipEditionCheck: $false, using root module: $false, using absolute path: $false, CompatiblePSEditions: $n
```powershell
Expected 0, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 493
493:                 $modules.Count | Should -Be 0
```
Get-Module nested module behaviour with Edition checking.Modules ON the System32 test path.Get-Module -ListAvailable gets all compatible modules when SkipEditionCheck: $false, using root module: $false, using absolute path: $false, CompatiblePSEditions: De
```powershell
Expected 0, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 493
493:                 $modules.Count | Should -Be 0
```